Shared/DataTransferObjects/AttendanceForCreationDto.cs

```diff
@@ -0,0 +1,7 @@
+namespace Shared.DataTransferObjects;
+
+public record AttendanceForCreationDto(
+    Guid EmployeeId,
+    DateTime CheckInTime,
+    string Status,
```

Comment:
Consider using an enum type or a validated value for the `Status` field instead of a raw string. This helps prevent invalid or inconsistent status values across the codebase.

```suggestion
StatusEnum Status
```